db/migrate/20230328231118_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: {to_table: :users}
```

Comment:
- [ ] you really did a great job. Please as you did with posts please make sure to have `author` as a reference to `user` i.e
`t.references :author, null: false, foreign_key: {to_table: :users}`